tools/react.puml

```diff
@@ -0,0 +1,76 @@
+@startuml
+sprite $react [64x64/16] {
+EEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE
```

Comment:
This file does not look right. Did you use an input picture with alpha-channels? The background is supposed to be white/`00` but totally off, almost black (even if it's `EE`)
![grafik](https://user-images.githubusercontent.com/6781173/117304694-4f087a00-ae7e-11eb-9938-782ebd437635.png)

Reply:
Same here